### PR TITLE
*: update run integration_test doc

### PR DESCRIPTION
**Pull request by Ehco1996**
### What problem does this PR solve? 
- None

### What is changed and how it works?

Currently we cannot dynamically change the configuration by setting environment variables in integration tests case,
beacuse we have lot's of hard code in yaml conf file , for examle: [source1.yaml](https://github.com/pingcap/dm/blob/master/tests/all_mode/conf/source1.yaml)

### Check List 

Tests 

 - No code

Code changes
- None

Side effects
- None

Related changes
 - Need to cherry-pick to the release branch

### CODE REVIEW

**Review by lichunzhu — COMMENTED** [review]
/lgtm

File: tests/README.md [hunk 1]
```diff
@@ -35,12 +35,12 @@
 
 1. Run `make dm_integration_test_build` to generate DM related binary for integration test
 
-2. Setup two MySQL servers (the first one: 5.6 ~ 5.7; the second one: 8.0) with [binlog enabled first](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html) and [set `GTID_MODE=ON`](https://dev.mysql.com/doc/refman/5.7/en/replication-mode-change-online-enable-gtids.html), export proper environment variables. Variables and their default values are showed in following table.
+2. Setup two MySQL servers (the first one: 5.6 ~ 5.7; the second one: 8.0) with [binlog enabled first](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html) and [set `GTID_MODE=ON`](https://dev.mysql.com/doc/refman/5.7/en/replication-mode-change-online-enable-gtids.html), You need set the mysql port and root password according to the following table.
 
     | MySQL | Host | Port| PASSWORD |
     | :------------ | :---------- | :------ | :---- |
-    | MySQL1 | `MYSQL_HOST1` / `127.0.0.1` | `MYSQL_PORT1` / `3306` | `MYSQL_PASSWORD1` / `123456` |
-    | MySQL2 | `MYSQL_HOST2` / `127.0.0.1` | `MYSQL_PORT2` / `3307` | `MYSQL_PASSWORD2` / `123456` |
```
Comment (GMHDBJD) [inline tests/README.md:43]: The variable not working? 🤔

Comment (Ehco1996) [reply tests/README.md:43]: yes,beacuse we have lot's of hard code in yaml conf file like [source1.yaml](https://github.com/pingcap/dm/blob/master/tests/all_mode/conf/source1.yaml)

**Review by GMHDBJD — APPROVED** [review]
/lgtm

**Pull request merged**